index.js

```diff
@@ -141,6 +144,13 @@ File.prototype.inspect = function() {
   return '<File ' + inspect.join(' ') + '>';
 };
 
+File.isCustomProp = function(key) {
+  var builtInFields = [
```

Comment:
I had it like that originally, but wanted it to be part of the method so you don't have to go looking for that array, and was under the impression that JS engines optimize stuff like that out with no performance hit. No problem moving it to the top thought.